instrumentation/metric/registry.go

```diff
@@ -76,6 +77,17 @@ type inMemoryRegistry struct {
 	}
 }
 
+func (r *inMemoryRegistry) Get(metricName string) metric {
+	r.mu.Lock()
+	defer r.mu.Unlock()
+	for _, metric := range r.mu.metrics {
```

Comment:
Agree, but since this pattern was also used in `.Remove()` and `.register()`, and since I don't see a reason for an intensive usage, I decided to keep it simple and use this pattern here as well.